Makefile

```diff
@@ -27,8 +27,14 @@ endif
 include erlang.mk
 include erlang.install.mk
 
+define newline
+
+
+endef
+export newline
```

Comment:
macOS's `sed` uses a relatively fussy syntax for the `i` command - see https://stackoverflow.com/a/14846357/4318200. We need literal newlines within the command.

Makefiles won't add literal newlines into your command without help from your shell. See https://stackoverflow.com/questions/649246/is-it-possible-to-create-a-multi-line-string-variable-in-a-makefile for details.